src/workloads/query/multiplanner/BlockingPlanWin.yml

```diff
@@ -0,0 +1,219 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  The goal of this test is to exercise multiplanning. The optimal plan is to employ a blocking plan 
+  by scanning a segment of empty data and conducting a blocking-sort operation, whereas the other 
+  plans' index provides the right sort order, but requires a full scan, and every document is 
+  rejected after the FETCH stage. Because the SBE multiplanner can't round-robin, it has a heuristic 
+  "try nonblocking plans first".  This scenario is a worst case for that heuristic, because we'll try 
+  the best plan last.
+
+  We expect classic to have better latency and throughput than SBE on this workload,
+  and we expect the combination of classic planner + SBE execution (PM-3591) to perform about
+  as well as classic.
+
+GlobalDefaults:
+  dbname: &db test
+  # Collection name used for queries.
+  coll: &coll Collection0
+
+  docCount: &docCount 1e5
+
+  maxPhase: &maxPhase 6
+  queryRepeats: &queryRepeats 1000
+
+Actors:
+- Name: DropCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *maxPhase
+      PhaseConfig:
+        Repeat: 1
+        Database: *db
+        Operations:
+        - OperationName: RunCommand
+          OperationCommand:
+            drop: *coll
+
+- Name: SetQueryKnobs
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: QueryKnobTemplate
+    Parameters:
+        # It's ok for this to run in parallel with 'DropCollection'.
+        # It has to be a separate actor because they target different DBs.
+      active: [0]
+      nopInPhasesUpTo: *maxPhase
+      collection: *coll
+
+- Name: CreateDataset
+  LoadConfig:
+    Path: "../../../phases/query/Multiplanner.yml"
+    Key: InsertTemplate
+    Parameters:
+      database: *db
+      threads: 1
+      active: [1]
+      nopInPhasesUpTo: *maxPhase
+      repeat: 1
+      collection: *coll
+      docCount: *docCount
+      document: {
+        # Genny's random generators appear to be repeatable, so we might as well
+        # make '_id' explicit to make the overall dataset repeatable. Omitting
+        # '_id' would make it auto-generated based on a timestamp.
+        _id: {^Inc: {start: 0}},
```

Comment:
Do we set incremental ids for all workloads?
Than we should update the Loader in our phases file to MonotonicLoader - it will do exactly that.
Also, with it we can use several threads to create data (like 4) to speed up dataset creation.